CHANGELOG.rst

```diff
@@ -3,42 +3,46 @@
 Changelog
 =========
 
+0.12.9
+------
+- Example Sanic integration along with register_tortoise hook in contrib (#163)
+
 0.12.7
 ------
-* Support connecting to PostgreSQL via Unix domain socket (simple case).
-* Self-referential Foreign and Many-to-Many keys are now allowed
+- Support connecting to PostgreSQL via Unix domain socket (simple case).
+- Self-referential Foreign and Many-to-Many keys are now allowed
```

Comment:
Why did you replace the stars with dashes?

Reply:
I just saw both stars and dashes being used and stars being used much less.
Sorry, I can undo that. I was just going for consistency

Reply:
I don't think it matters at all for rst.
Could you keep the churn to a minimum?